StickySessions/Extensions/UITextViewPH.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  UITextViewPH.swift
+//  StickySessions
+//
+//  Created by Vinicius Albuquerque on 14/11/20.
+//  Copyright © 2020 CESAR. All rights reserved.
+//
+
+import UIKit
+
+class UITextViewPH: UITextView, UITextViewDelegate {
+    
+    var isPlaceholderShowing = true
+    private var phText = "Add text here"
```

Comment:
Is this "**ph**Text" a convention? Can we give a more meaningful name for this variable?